docs/style.css

```diff
@@ -0,0 +1,187 @@
+.custom-theme {
+    --cds-background-brand: #6929c4;
+    --cds-border-interactive: #6929c4;
+    --cds-focus: #6929c4;
+    --cds-icon-interactive: #6929c4;
+    --cds-interactive: #6929c4;
+    --cds-link-primary: #6929c4;
+    --cds-button-primary: #6929c4;
+    --cds-button-tertiary: #6929c4;
+    --cds-background-brand: #6929c4;
+    --cds-border-interactive: #6929c4;
+    --cds-focus: #6929c4;
+    --cds-icon-interactive: #6929c4;
+    --cds-interactive: #6929c4;
+    --cds-link-primary: #6929c4;
+    --cds-button-primary: #6929c4;
+    --cds-button-tertiary: #6929c4;
+    --cds-background-brand: #6929c4;
+    --cds-focus-inverse: #6929c4;
+    --cds-link-inverse: #6929c4;
+    --cds-button-primary: #6929c4;
+    --cds-notification-action-tertiary-inverse: #6929c4;
+    --cds-background-brand: #6929c4;
+    --cds-focus-inverse: #6929c4;
+    --cds-link-inverse: #6929c4;
+    --cds-button-primary: #6929c4;
+    --cds-notification-action-tertiary-inverse: #6929c4;
+  
+    --cds-button-primary-hover: #491d8b;
+    --cds-button-tertiary-hover: #491d8b;
+    --cds-button-primary-hover: #491d8b;
+    --cds-button-tertiary-hover: #491d8b;
+    --cds-button-primary-hover: #491d8b;
+    --cds-notification-action-tertiary-inverse-hover: #491d8b;
+    --cds-button-primary-hover: #491d8b;
+    --cds-notification-action-tertiary-inverse-hover: #491d8b;
+  
+    --cds-link-primary-hover: #491d8b;
+    --cds-link-secondary: #491d8b;
+    --cds-support-info: #491d8b;
+    --cds-link-primary-hover: #491d8b;
+    --cds-link-secondary: #491d8b;
+    --cds-support-info: #491d8b;
+    --cds-highlight: #491d8b;
+    --cds-link-inverse-hover: #491d8b;
+    --cds-support-info-inverse: #491d8b;
+    --cds-tag-background-blue: #491d8b;
+    --cds-link-inverse-hover: #491d8b;
+    --cds-support-info-inverse: #491d8b;
+    --cds-tag-background-blue: #491d8b;
+   }
+
+*{
+    box-sizing: inherit;
+}
+
+html, body {
+    height: 100%;
+    margin: 0;
+    background-color: #f4f4f4;
+}
+
+body {
+    flex-flow: column nowrap;
+    font-family: 'IBM Plex Sans', sans-serif;
+}
+
+header{
+    padding: 20px 0px 20px 0px;
+    background-color: white;
+}
+
+.content{
+    padding: 20px 30px 20px 30px;
+    margin-bottom: 15px;
+}
+
+.cards{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 30px;
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
+.card{
+    flex: 0 1 20%;
+    min-width: 300px;
+    background-color: white;
+    /*height: 100%; /*Cards have the necessary height*/
```

Comment:
is this commented out code necessary? can we remove it or are you keeping it there for a specific reason?